### DISCUSSION TITLE
How to install Pimcore 10.x using docker images

### DISCUSSION THREAD

**Discussion by san4ever** [pimcore/pimcore#9978] [Q&A / Support] [answered]
i have gone through the documentation. I need the docker files to install the pimcore set up so i can deploy in Amazon EKS cluster

**Comment by alexej-d** [comment, marked as answer]
You can use [this](https://github.com/pimcore/skeleton) or [this](https://github.com/pimcore/demo) repository's docker-compose.yml

**Reply by san4ever** [reply]
Hi @alexej-d 
Thanks for replying. I am looking for docker files to install pimcore with **nginx**. Is it alreay available ?
